app/server/api/routes.py

```
@@ -0,0 +1,287 @@
+        # Get basic file/folder structure
+        structure = repo_analyzer.analyze_repo(repo_path)
+
+        # Parse code files to extract functions and classes
+        code_nodes = []
+        for node in structure.nodes:
+            if hasattr(node, 'extension') and node.extension in ['.py', '.js', '.jsx', '.ts', '.tsx']:
+                try:
+                    parsed_nodes = code_parser.parse_file(node.path)
+                    code_nodes.extend(parsed_nodes)
+                except Exception:
+                    continue
```

Ruff (0.14.1)

`try`-`except`-`continue` detected, consider logging the exception

(S112)